sw/airborne/modules/computer_vision/opencv_image_functions.h

```diff
@@ -0,0 +1,9 @@
+
+#ifndef OPENCV_IMAGE_FUNCTIONS_H
+#define OPENCV_IMAGE_FUNCTIONS_H
+#include <opencv2/core/core.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+using namespace cv;
```

Comment:
Also it's good style to not use namespaces in the header files, instead use `cv::Mat` below.